main.go

```diff
@@ -105,6 +110,7 @@ func main() {
 	api.HandleFunc("/{account}/buckets/{bucket}", BucketHeadHandler).Methods(http.MethodHead)
 	// api.HandleFunc("/{account}/buckets/{bucket}", BucketShowHandler).Methods(http.MethodGet)
 	api.HandleFunc("/{account}/buckets/{bucket}", BucketDeleteHandler).Methods(http.MethodDelete)
+	api.HandleFunc("/{account}/buckets/{bucket}/objects", ObjectCountHandler).Methods(http.MethodHead)
```

Comment:
Does this handler exist?

Reply:
no it probably doesn't, i can fix this once its merged into the other branch